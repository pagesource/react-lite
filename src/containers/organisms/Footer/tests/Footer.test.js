import React from 'react';
import { shallow } from 'enzyme';

import MockProvider from '../../../../../__mocks__/mockProvider';

import Footer from '../index';

describe('Footer component', () => {
  test('should render correctly', () => {
    const FooterComponent = shallow(
      <MockProvider>
        <Footer />
      </MockProvider>
    );
    expect(FooterComponent).toMatchSnapshot();
  });
});
