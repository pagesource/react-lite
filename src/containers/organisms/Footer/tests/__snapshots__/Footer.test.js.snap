// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <MemoryRouter>
    <Connect(Footer) />
  </MemoryRouter>
</Provider>
`;
